cli/cmd/install.go

```diff
@@ -105,7 +105,7 @@ control plane.`,
   linkerd install | kubectl apply -f -
 
 The installation can be configured by using the --set, --values, --set-string and --set-file flags.
-A full list of configurable values can be found at https://www.github.com/linkerd/linkerd2/tree/main/charts/linkerd2/README.md`,
+A full list of configurable values can be found at https://www.artifacthub.io/packages/helm/linkerd2/linkerd2#values`,
```

Comment:
That link isn't working for me, and it targets the old chart.  Should this be https://artifacthub.io/packages/helm/linkerd2-edge/linkerd-control-plane ?

Reply:
should we have some logic to conditionally set `linkerd2-edge` vs `linkerd2` depending on if this is a stable build?

Reply:
Oh weird I added `www` after copying the URL but it doesn't like that; I'll remove it. We could point it to the edge but it seems like a better idea to point it to stable for now.

Reply:
The weird thing about that is the linkerd-control-plane chart hasn't had a stable release yet.  Which means that if we link to https://artifacthub.io/packages/helm/linkerd2/linkerd-control-plane there's nothing there yet.  So all the edge releases until the stable will have a broken link.  And, going forward, as we add new values in future edges, they won't be reflected in the stable docs.